src/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -2090,13 +2090,13 @@ msgstr "Current Review Stage: %(review_stage)s"
 
 #: templates/default/reviews/talk_proposal_list.html:18
 msgid ""
-"Note that at anytime, the identity of the reviewer, e.g. you, is hidden to "
-"other reviewers and speakers. For more information about the whole review "
+"Please note that the identities of all reviewers are always invisible to "
+"other reviewers and submitters. For more information about the whole review "
 "process, please refer to the GitBook <a href=\"https://pycontw.github.io/"
 "reviewer-guidebook\" target=\"_blank\" rel=\"noopener\">Review Guideline</a>."
 msgstr ""
-"Note that at anytime, the identity of the reviewer, e.g. you, is hidden to "
-"other reviewers and speakers. For more information about the whole review "
+"Please note that the identities of all reviewers are always invisible to "
+"other reviewers and submitters. For more information about the whole review "
```

Comment:
**issue (blocking):** these update should be changed from running `python3 manage.py makemessages` so that the i18n could work properly.

Reply:
solved by `dc91ecc`